calculatorFrame.java

```diff
@@ -111,191 +104,192 @@ public void init() {
 				System.out.println(firstNumString);
 				firstNumString += "5";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "5";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Six button
-		else if (e.getSource() == six) {
-			if (flag == false){
+		else if(e.getSource() == six) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "6";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "6";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Seven button
-		else if (e.getSource() == seven) {
-			if (flag == false){
+		else if(e.getSource() == seven) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "7";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "7";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Eight button
-		else if (e.getSource() == eight) {
-			if (flag == false){
+		else if(e.getSource() == eight) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "8";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "8";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Nine button
-		else if (e.getSource() == nine) {
-			if (flag == false){
+		else if(e.getSource() == nine) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "9";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			}else if(flag == true) {
 				secondNumString += "9";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//Zero button
-		else if (e.getSource() == zero){
-			if (flag == false){
+		else if(e.getSource() == zero) {
+			if(flag == false) {
 				System.out.println(firstNumString);
 				firstNumString += "0";
 				System.out.println(firstNumString);
 				textArea.setText(textArea.getText() + e.getActionCommand());
-			}else if(flag == true){
+			} else if(flag == true) {
 				secondNumString += "0";
 				duplicateOperatorFlag = false;
 				textArea.setText(textArea.getText() + e.getActionCommand());
 			}
 		}
-		
+
 		//DECIMAL
```

Comment:
You can remove this comment and code since it has the same behaviour as the number buttons